app/assets/javascripts/models/story.js

```diff
@@ -39,7 +39,7 @@ var Story = module.exports = Backbone.Model.extend({
   },
 
   changeState: function(model, new_value) {
-    if (new_value == "started" && !this.get('owned_by_id')) {
+    if (new_value == "started") {
```

Comment:
Can we rename `new_value` to `newValue`? I know you weren't the one that added the whole function but it seems short enough to be easy to rename it and enforce the code styling we're using. Also, it's better to use `===` instead of `==`.